ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/views/ManageSubscriptionsView.kt

```diff
@@ -172,11 +179,18 @@ private fun ContentUnavailableView(
 @Composable
 private fun ManageSubscriptionsButtonsView(
     screen: CustomerCenterConfigData.Screen,
+    purchaseInformation: PurchaseInformation?,
     onButtonPress: (CustomerCenterConfigData.HelpPath) -> Unit,
 ) {
+    val filteredPaths = if (purchaseInformation?.isLifetime == true) {
+        screen.supportedPaths.filter { it.type != CustomerCenterConfigData.HelpPath.PathType.CANCEL }
+    } else {
+        screen.supportedPaths
+    }
```

Comment:
hmm, this is the View, it feels like this logic should live in the view model instead

Reply:
Discussed offline
I'll introduce something else (also good for testing), keeping in mind having only one view model